### ISSUE TITLE
Test that there is always a selected node if there are open reports

### ISSUE THREAD

**Issue by mhdirkse** [ibissource/ladybug-frontend#58]
* Create two reports, names “name” and “otherName”.
* Open Ladybug.
* In the table, click the report with name “name”.
* Check that there are three nodes with name “name”.
* Check that only one node has class “node-selected”.
* In the table, click the report with name “otherName”.
* Check that there are six nodes.
* Check that the selected node has name “name”.
* Close the current report (close button to the right).
* Check that all remaining nodes have name “otherName”.
* Check that there is one selected node.

**Comment by jacodg** [comment]
Yes, I agree that as user when you select a report in the table you would expect it to "open" and that the first node of that report is automatically selected
